docs/how-to/design-tokens-support.rst

```diff
@@ -0,0 +1,189 @@
+For example, if you were working with Paragon v23.0.0 you would navigate to https://github.com/openedx/paragon/tree/v23.0.0/tokens.
+
+.. code-block::
+  
+    paragon/
+    └── tokens/
+        └── src/
+            ├── core/
+            │   └── <name_of_the_folder>/
+            │       └── <name_of_the_file>.json
+            └── themes/
+                ├── light/
+                │   └── <name_of_the_folder>/
+                │       └── <name_of_the_file>.json
+                ├── dark/
+                │   └── <name_of_the_folder>/
+                │       └── <name_of_the_file>.json
+                └── my-theme/
+                    └── <name_of_the_folder>/
+                        └── <name_of_the_file>.json
+
+
+Once you have identified the token to override, you can replace ``<name_of_the_folder>`` and ``<name_of_the_file>`` with the right names.
+Currently, Paragon is organized under **semantic tokens**, for a better understanding visit: 
+`Design tokens implementation in Paragon <https://github.com/openedx/paragon/blob/master/docs/decisions/0019-scaling-styles-with-design-tokens.rst#design-tokens-implementation-in-paragon>`
+
+Tokens format
+=============
+
+In terms of tokens, Paragon follows the specifications of the `Design Tokens Community Group <https://tr.designtokens.org/format/#abstract>`_, and
+takes `style dictionary token structure <https://styledictionary.com/info/tokens/#category--type--item>`_ as inspiration:
+
+
+.. image:: ./style_diccionary_tokens.webp
+
+
+With a subtle variation the structure to define most of the Paragon tokens is ``category > item > subitem > type > state``, for example:
+
+.. code-block:: json
+  
+    {
+      "spacing": {    // Category
+        "$type": "dimension",                         
+        "annotation": {    // Item 
+          "padding": {     // Type
+              "$value": ".5rem",
+              "$source": "$annotation-padding"
+          },
+          "arrow-side": {   // Subitem
+            "margin": {     // Type
+              "$value": "{spacing.annotation.padding}",
+              "$source": "$annotation-arrow-side-margin"
+            }
+          }
+        }
+      },
+      "typography": {
+        "annotation": {
+          "font-size": { 
+          "source": "$annotation-font-size", 
+          "$value": "{typography.font.size.sm}", 
+          "$type": "dimension" 
+          },
+        }
+      },
+    }
+
+Each token has specific attributes:
+
+- **Value**: It is the value that will be assigned to the variable, which could be a value or a reference, such as l arrow-side in the above example.
+- **Type**: Indicates the property to be processed (color, dimension, etc..). This value could be defined for the token itself or a group of tokens (e.g. spacing)
+- **Source**: This value is additional and indicates the equivalent in saas notation.

```

Comment:
[consider] I might suggest not mentioning `source` in the brand package, as it's mostly relevant for the core Paragon tokens, not the brand token overrides. For example, if a brand package overrides an existing `@openedx/paragon` token that itself uses `source`, the token in the brand package will still have / inherit the original `source`.

Reply:
The main reason why I left source in the explanation was the comment bellow. However, I'm not against the recommendation. 

https://github.com/openedx/brand-openedx/pull/26/files#diff-26010d3309a977df1bd197514007774722f7f0afde99a35b868424ed6f034ef1R88